### PR TITLE
docs: native binaries installation docs

### PR DESCRIPTION
**Pull request by negar-abbasi**
Fixes [#436](https://github.com/algorandfoundation/algokit-cli/issues/436)

## Proposed Changes

  - update docs based on native binary installation methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,50 +106,30 @@ AlgoKit can be installed using OS specific package managers, or using the python
 
 ## Install AlgoKit on Windows
 
-> **Note**
-> This method will install the most recent python3 version [via winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/). If you already have python 3.10+ installed, you may [prefer to use pipx directly instead](#install-algokit-with-pipx-on-any-os) so you can control the python version used.
-
 1. Ensure prerequisites are installed
 
+   - [winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/)
```
Comment (neilcampbell) [inline README.md:111]:
```suggestion
   - [WinGet](https://learn.microsoft.com/en-us/windows/package-manager/winget/) (should be installed by default on recent Windows 10 or later)
```

File: README.md [hunk 2]
```diff
@@ -104,50 +106,30 @@ AlgoKit can be installed using OS specific package managers, or using the python
...
-   3. Install pipx:
-      ```
-      pip install --user pipx
-      python -m pipx ensurepath
-      ```
-   4. Restart the terminal to ensure pipx is available on the path
-   5. Install AlgoKit via pipx: `pipx install algokit`
-   6. Restart the terminal to ensure AlgoKit is available on the path
+     
+2. Install using winget
```
Comment (neilcampbell) [inline README.md:117]:
```suggestion
2. Install using WinGet
```

File: README.md [hunk 3]
```diff
@@ -104,50 +106,30 @@ AlgoKit can be installed using OS specific package managers, or using the python
...
 
 3. [Verify installation](#verify-installation)
 
 ### Maintenance
 
 Some useful commands for updating or removing AlgoKit in the future.
 
-- To update AlgoKit: `pipx upgrade algokit`
-- To remove AlgoKit: `pipx uninstall algokit`
+- To update AlgoKit: `winget upgrade AlgoKit`
```
Comment (neilcampbell) [inline README.md:128]:
```suggestion
- To update AlgoKit: `winget upgrade algokit`
```

File: README.md [hunk 4]
```diff
@@ -104,50 +106,30 @@ AlgoKit can be installed using OS specific package managers, or using the python
...
+- To remove AlgoKit: `winget uninstall AlgoKit`
```
Comment (neilcampbell) [inline README.md:129]:
```suggestion
- To remove AlgoKit: `winget uninstall algokit`
```

File: README.md [hunk 5]
```diff
@@ -171,16 +156,22 @@ Some useful commands for updating or removing AlgoKit in the future.
 
 1. Ensure prerequisites are installed
 
-   - [Python 3.10+](https://www.python.org/downloads/)
-
-     > **Note**
-     > There is probably a better way to install Python than to download it directly, e.g. your local Linux package manager
-
-   - [pipx](https://pypa.github.io/pipx/#on-linux-install-via-pip-requires-pip-190-or-later)
+   - [Snap](https://snapcraft.io/docs/installing-snapd) (should already be available on Ubuntu 16.04.4 LTS (Xenial Xerus) or later)
```
Comment (neilcampbell) [inline README.md:159]:
```suggestion
   - [Snap](https://snapcraft.io/docs/installing-snapd) (should be installed by default on Ubuntu 16.04.4 LTS (Xenial Xerus) or later)
```

File: README.md [hunk 6]
```diff
@@ -83,19 +83,21 @@ This is an open source project managed by the Algorand Foundation. See the [cont
...
 > **Note**
-> You can still install and use AlgoKit without these dependencies and AlgoKit will tell you if you are missing one for a given command.
+> You can still install and use AlgoKit without these dependencies, and AlgoKit will tell you if you are missing one for a given command.
 
 - Git - Git is used when creating and updating projects from templates
 - Docker - Docker Compose (and by association, Docker) is used to run the AlgoKit LocalNet environment, we require Docker Compose 2.5.0+
 
+> **Note**
+> If you are using pipx to install AlgoKit, the key required dependency is Python 3.10+.
+
```
Comment (neilcampbell) [inline README.md:96]:
```suggestion
```

We already have this listed as a prerequisites in the pipx install instructions, so not need to have this note

File: README.md [hunk 7]
```diff
@@ -83,19 +83,21 @@ This is an open source project managed by the Algorand Foundation. See the [cont
...
+
```
Comment (neilcampbell) [inline README.md:97]: Maybe we should add a note, saying if you have previously installed AlgoKit using pipx, however would like to switch to using one of the other installations method, please uninstall using `pipx uninstall algokit`, before installing using the instructions for your desired platform.

File: README.md [hunk 8]
```diff
@@ -83,19 +83,21 @@ This is an open source project managed by the Algorand Foundation. See the [cont
...
 ## Cross-platform installation
 
-AlgoKit can be installed using OS specific package managers, or using the python tool [pipx](https://pypa.github.io/pipx/) see below for specific installation instructions.
+AlgoKit can be installed using OS specific package managers, or using the python tool [pipx](https://pypa.github.io/pipx/). See below for specific installation instructions.
```
Comment (neilcampbell) [inline README.md:100]: Should the installation methods document the processor architectures we support?
One thing I did notice for windows was the binary built on the GitHub agent, seemed to work fine on my Windows ARM VM.

Comment (negar-abbasi) [reply README.md]: the processor architectures we support are now documented. I stick to not supporting Windows ARM for now. If we think there is no issue supporting that then maybe we also need to change its release artifact name as well.

File: README.md [hunk 9]
```diff
@@ -83,71 +83,62 @@ This is an open source project managed by the Algorand Foundation. See the [cont
 
 ## Prerequisites
 
-The key required dependency is Python 3.10+, but some of the installation options below will install that for you.
-
-AlgoKit also has some runtime dependencies that also need to be available for particular commands.
+AlgoKit has some runtime dependencies that also need to be available for particular commands.
```
Comment (neilcampbell) [inline README.md:86]:
```suggestion
Your specific installation method documents the prerequisites for running AlgoKit. Additionally AlgoKit has some runtime dependencies that also need to be available for particular commands.
```

File: README.md [hunk 10]
```diff
@@ -83,71 +83,62 @@ This is an open source project managed by the Algorand Foundation. See the [cont
...
 ## Cross-platform installation
 
-AlgoKit can be installed using OS specific package managers, or using the python tool [pipx](https://pypa.github.io/pipx/) see below for specific installation instructions.
+AlgoKit can be installed using OS specific package managers, or using the python tool [pipx](https://pypa.github.io/pipx/). 
+See below for specific installation instructions and supported architectures.
 
+### Supported Operating Systems and Architectures
+
+- **macOS**: x64 and ARM (Apple Silicon)
+- **Windows**: Windows 10 and later (x64 only; ARM not supported)
```
Comment (neilcampbell) [inline README.md:107]: WinGet is only support on `Windows 10 1709 (build 16299)` and later.

**Review by neilcampbell — APPROVED** [review]

**Pull request merged**